docs/runtime-contract.md

```diff
@@ -211,9 +211,9 @@ following protocol will be used:
 - `h2c`: HTTP/2 transport, as described in
   [section 3.4 of the HTTP2 spec (Starting HTTP/2 with Prior Knowledge)](https://http2.github.io/http2-spec/#known-http)
 
-Developers SHOULD prefer to use automatic content negotiation where available,
+Developers SHOULD use automatic content negotiation where available,
 and MUST NOT set the `name` field to arbitrary values, as additional transports
-may be defined in the future. Developers MUST assume all traffic is
+might be defined in the future. Developers MUST assume all traffic is
```

Comment:
The "Developers MUST assume" and "Developers MUST NOT assume" should use non-keywords.

Reply:
done